app/templates/partials/introduction/basic.html

```diff
@@ -7,7 +7,7 @@ <h2 class="saturn">{{ intro.title }}</h2>
     <h3 class="venus">{{ content.title }}</h3>
     {% endif %}
     {% if content.description %}
-    <p class="mars qa-intro-description">
+    <p class="mars qa-intro-description" data-qa="intro-description">
```

Comment:
If you're going to use these then it's still best to uniquely name them. Currently same as intro-description on preview.html. `into-basic-description` and `intro-preview-description`???